test/igniter/project/formatter_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule Igniter.Project.FormatterTest do
+  use ExUnit.Case
+  import Igniter.Test
+
+  describe "import_dep/2" do
+    test "regression: causes formatting to respect imported locals_without_parens" do
+      test_project(
+        files: %{
+          "test/formatter_test.exs" => """
+          defmodule FormatterTest do
+            use ExUnit.Case
+            use Mimic.DSL
+
+            test "1" do
+              expect Foo.add(x, y), do: x + y
+            end
+          end
+          """
+        }
+      )
+      |> Igniter.Project.Deps.add_dep({:mimic, "~> 1.7"})
+      |> Igniter.Project.Formatter.import_dep(:mimic)
```

Comment:
This is a little hacky, but I needed a dep we already had that exports some locals_without_parens, and Mimic was the one I found.